CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 - [Read all properties of ApplicationInsightsServiceOptions from IConfiguration](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1882)
 - [End support for NetStandard 1.x, Add support for NetStandard 2.0](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1160)
 - [Add support for SourceLink.Github to all SDKs.](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1760)
-
+- [Fix: ServerTelemetryChannel stores telemetry data in default folder during transient errors in all environments](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1792)
```

Comment:
Lets rewrite this to say something like:
ServerTelemetryChannel by default uses local disk storage in non Windows, to store telemetry during transient errors.